src/dataIntegration/transcriptomics/mapGeneToRxn.m

```diff
@@ -0,0 +1,64 @@
+function expressionRxns = mapGeneToRxn(model, gene_id, gene_expr, parsedGPR, corrRxn)

```

Comment:
In the code, I'm using the rule of :AND is replace by MIN and OR by max following Becker SA, Palsson BO. Context-specific metabolic networks are consistent with experiments. PLoS Comput Biol. 2008;4:e1000082. doi: 10.1371/journal.pcbi.1000082. (also applied in mCADRE)